app/cli/internal/action/workflow_run_list.go

```diff
@@ -115,6 +115,8 @@ func humanizedRunnerType(in v1.CraftingSchema_Runner_RunnerType) string {
 		return "GitHub"
 	case *v1.CraftingSchema_Runner_GITLAB_PIPELINE.Enum():
 		return "GitLab"
+	case *v1.CraftingSchema_Runner_AZURE_DEVOPS_PIPELINE.Enum():
+		return "Azure DevOps Pipeline"
```

Comment:
This information I believe will be shown in the tables, I am wondering if it's too long compared to the others.

In fact, [here](https://azure.microsoft.com/en-us/products/devops/pipelines#overview) they seem to reference to it as `Azure pipelines`, would it make sense to do the same here?

Reply:
> in fact, [here](https://azure.microsoft.com/en-us/products/devops/pipelines#overview) they seem to reference to it as Azure pipelines

Yes, I found both references and was already thinking about making it shorter. Thanks, I will do it.

Reply:
- <img width="430" alt="Screenshot 2023-07-29 at 23 06 51" src="https://github.com/chainloop-dev/chainloop/assets/4465/f75b31a1-dd41-4cf8-a7eb-48de571a1ea0">
- <img width="287" alt="Screenshot 2023-07-29 at 23 06 16" src="https://github.com/chainloop-dev/chainloop/assets/4465/9ebbf29b-3ac7-41ab-847c-5b87745f6ecd">